com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineMaterialUpgrader.cs

```diff
@@ -54,7 +54,7 @@ private static void GetShaderNamesToIgnore(ref HashSet<string> shadersToIgnore)
             shadersToIgnore.Add("Sprites/Default");
         }
 
-        private static void GetUpgraders(ref List<MaterialUpgrader> upgraders)
+        internal static void GetUpgraders(ref List<MaterialUpgrader> upgraders)
```

Comment:
Graphics: Please note private API made internal here

Reply:
If I understand correctly this change will not be needed anymore as the plan is to integrate the AnimationClipUpgrader with the existing material upgrader.